recipe/meta.yaml

```diff
@@ -95,6 +94,8 @@ test:
   commands:
     - env with_test_suite={{ with_test_suite }} bash test_openmm.sh   # [unix]
     - set with_test_suite={{ with_test_suite }} && .\test_openmm.bat  # [win]
+    - env with_test_suite={{ with_test_suite }} bash test_openmm.sh   # [osx]
```

Comment:
I think this line might be redundant... The `unix` selector two lines above is an alias for `linux or osx`.